packages/apps-engine/tests/test-data/bridges/roomBridge.ts

```
@@ -1,8 +1,10 @@
 import type { IMessage, IMessageRaw } from '../../../src/definition/messages';
 import type { IRoom } from '../../../src/definition/rooms';
+import type { IRoomRaw } from '../../../src/definition/rooms/IRoomRaw';
+import type { IRoomRaw } from '../../../src/definition/rooms/IRoomRaw';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'IRoomRaw'. Consider to delete it or rename it.

'IRoomRaw' is defined here:


(lint/suspicious/noRedeclare)